Eli's branch

generalripphook opened the pull request:
Character Sheet changes to weapon profiles 
Fumbles, Criticals with hm_std

generalripphook committed:
Combat Profile Overhaul

generalripphook committed:
Checkbox Update

generalripphook committed:
Create API Criticals

generalripphook committed:
Create API Melee Fumble

generalripphook committed:
Create API Ranged Fumbles

t0q reviewed (changes requested):
Looks fine to me, assuming it works out. But you've got this thing going where you're introducing blank lines everywhere. Fix that and we're golden.

generalripphook committed:
Formatting, Opposed Skill Dice

I removed the extra spaces, and I have added a check box for opposed dice but it only currently works for acting
